database-mysql/index.js

```diff
@@ -1 +1,33 @@
 // this is where the database connection will happen 
+const Sequelize = require('sequelize');
```

Comment:
when adding a package like sequelize, make sure to install it with --save so it save it as a dependency in the package.json file, then commit your package.json file if you've changed it